consoles/video_stream.pm

```diff
@@ -0,0 +1,380 @@
+# Copyright 2021 Marek Marczykowski-Górecki
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package consoles::video_stream;
+
+use Mojo::Base -strict, -signatures;
+use feature 'say';
+
+use base 'consoles::network_console';
+
+use List::Util 'max';
+use Time::HiRes qw(usleep);
+
+use Try::Tiny;
+use testapi 'get_var';
+use bmwqemu qw(diag);
+
+# speed limit: 30 keys per second
+use constant STREAM_TYPING_LIMIT_DEFAULT => 30;
+
+use constant DV_TIMINGS_CHECK_INTERVAL => 3;
+
+use constant STALL_THRESHOLD => 4;
+
+sub screen {
+    my ($self) = @_;
```

Comment:
You activate `-signatures` at the top, so you should probably use signatures in the methods. 😁